torch/csrc/jit/ir/alias_analysis.cpp

```diff
@@ -1488,5 +1521,30 @@ MemoryLocations AliasDb::buildWrittenToLocationsIndex() const {
   return ret;
 }
 
+void Lint(const AliasDb* db) {
+  bool failed = false;
+
+  std::stringstream ss;
+  // Every mutable value in the system has a corresponding element.
+  for (const auto& v : db->graph_->all_values) {
+    if (!db->isMutableTypeInternal(v)) {
+      continue;
+    }
+    auto it = db->elementMap_.find(v);
+    if (it == db->elementMap_.end()) {
+      failed = true;
+      ss << "Value %" << v->debugName() << " of type "
+         << v->type()->python_str() << " wasn't found in the element map.\n"
+         << "It was defined in " << *v->node();
+    }
+  }
+  TORCH_INTERNAL_ASSERT(!failed, ss.str());
+
+  // Two checks that we want to add but can't until the mutation API is more
+  // fully developed.
+  // - Every mutable value in the aliasdb belongs to the graph
+  // - All container values have contained elements
```

Comment:
I think we're not checking this bc we break this invariant as part of this PR.

Reply:
Yeah, the comment notes that it is not yet added pending a more robust mutation API.